extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -173,6 +173,11 @@ amp-carousel .amp-carousel-button.amp-disabled {
 .i-amphtml-slidescroll-no-snap .i-amphtml-slide-item {
   scroll-snap-coordinate: none !important;
 }
+
+.i-amphtml-slidescroll-no-snap.i-amphtml-slides-container.i-amphtml-no-scroll {
+  -webkit-overflow-scrolling: auto !important;
```

Comment:
it wont - this is only enabled when `.i-amphtml-no-scroll` class is present , which means our script already disabled scrollbar and took over the animation